### PR TITLE
[fix]: [PL-28861]: Addressed Review comments on API Docs

### PR DESCRIPTION
**Pull request by mankrit-singh**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/38591)

### CODE REVIEW

File: docs/api-standards/README.md [hunk 1]
```diff
@@ -9,11 +9,24 @@ Harness uses a Representational State Transfer (RESTful) style API to expose fun
 RESTful APIs are focused on resources, how to group them, and a set of actions that can be taken against them. A resource is a noun. Resources are grouped into collections and actions taken against them. Given RESTful APIs are generally built on top of the HTTP protocol, resources are mapped to URL endpoints and HTTP verbs are used to handle the default actions to be taken against a resource or collection.
 
 
-## Process For Change
+## Spec-first API development
```
Comment (allofthesepeople) [inline docs/api-standards/README.md:12]: I think this section warrants its own file. Please move to a new markdown file of the same name.

The original heading "Process for Change" was a placeholder for how to change this set of standards once the CAG team had decided upon one. Would be good to keep it in the README.

**Review by allofthesepeople — APPROVED** [review]
Looks great, thanks! 

Approving so as not to block but please see the comment on moving the "spec-first" section to a new file and re-adding the "process for change" heading.

**Review by ashishsanodia-harness — APPROVED** [review]
> Reviewed 6 of 7 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/38591)* status: 8 file(s) to review. 1 of 2 approvals obtained.

**Review by kshitij-harness — APPROVED** [review]
> Reviewed 6 of 7 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/38591)* status: :shipit: complete! ✓

**Pull request merged**